benchmarks/GraphBLAS-sharp.Benchmarks/MatrixExtensions.fs

```diff
@@ -0,0 +1,91 @@
+namespace GraphBLAS.FSharp.Benchmarks
+
+open GraphBLAS.FSharp.Objects
+open Brahma.FSharp
+open GraphBLAS.FSharp.Backend.Objects.ClMatrix
+
+module MatrixExtensions =
+    type Matrix<'a when 'a : struct> with
+        static member ToBackendCOO (context: ClContext) matrix =
```

Comment:
I was unable to do this because of the cyclic dependency between the definition of the `Matrix` and `Matrix.{MatrixFormat}` in this case...
https://github.com/YaccConstructor/GraphBLAS-sharp/blob/32820df025de7d31b2c3f7de225325d1ddcd7d4c/src/GraphBLAS-sharp/Objects/Matrix.fs#L148

Reply:
Well... Looks like a design issue. But anyway,  looks like you can just call `.toDevice` instead of manual conversion.